### PR TITLE
#3281 qtgui: matplotlib dependency

### PR DESCRIPTION
**Pull request by sidkapoor97**
refrences https://github.com/gnuradio/gnuradio/issues/3281

### CODE REVIEW

**Comment by mormj** [comment]
I commented on the issue, not the PR. Will add review comments

**Comment by mormj** [comment]
Also - @sidkapoor97 - thank you for taking a look into this!

**Comment by mormj** [comment]
@sidkapoor97 - this particular change will fall below the 10 line limit for needing a CLA (contributor license agreement).  If you plan on contributing larger changes in the future, consider getting that process started - see https://wiki.gnuradio.org/index.php/Development#What.27s_this_Copyright_Assignment.3F

**Comment by sidkapoor97** [comment]
Thank you, I will have a look into the copyrights and I believe that solves the PR

File: gr-qtgui/python/qtgui/__init__.py [hunk 1]
```diff
@@ -23,14 +23,22 @@
...
+try:
+	import matplotlib.pyplot as plt
+	from .distanceradar import DistanceRadar
+	from .azeplot import AzElPlot
+except ImportError:
+	from gnuradio import gr
+	gr.log.warn("Matplotlib is a required dependency to use DistanceRadar and AzElPlot."
+                "  Please install matplotlib to use these blocks (https://matplotlib.org/)")
+
+
```
Comment (mormj) [inline gr-qtgui/python/qtgui/__init__.py:35]: Not huge deal, but can you please remove extra line of whitespace

Comment (sidkapoor97) [reply gr-qtgui/python/qtgui/__init__.py]: Changes have been incorporated thank you :) @mormj for helping

**Comment by drmpeg** [comment]
I don't think this PR is working as expected. It prints the error message even when matplotlib is available.

image

**Comment by drmpeg** [comment]
It's a typo. azeplot should be azelplot.

**Comment by sidkapoor97** [comment]
I am sorry I don't know how it got missed, I will just correct it in a while

**Comment by sidkapoor97** [comment]
@mormj should we make a new PR for this?

**Comment by mormj** [comment]
Yes please do!

**Pull request merged**